Sources/Foundation/NSURL.swift

```diff
@@ -1444,8 +1450,8 @@ open class NSURLComponents: NSObject, NSCopying {
             return (0..<count).map { idx in
                 let oneEntry = unsafeBitCast(CFArrayGetValueAtIndex(queryArray, idx), to: NSDictionary.self)
                 let swiftEntry = oneEntry._swiftObject
-                let entryName = swiftEntry["name"] as! String
-                let entryValue = swiftEntry["value"] as? String
+                let entryName = swiftEntry[_kCFURLComponentsNameKey as! String] as! String
+                let entryValue = swiftEntry[_kCFURLComponentsValueKey as! String] as? String
```

Comment:
@millenomi could this change be causing the ci failure?
> Fatal error: Constant strings cannot be deallocated: file /home/buildnode/jenkins/workspace/swift-corelibs-foundation-PR-Linux/swift-corelibs-foundation/Sources/Foundation/NSCFString.swift, line 119

Backtrace:
```
0    libswiftCore.so                    0x00007f88d7c7cfb0 swift_reportError + 50
1    libswiftCore.so                    0x00007f88d7cefea0 _swift_stdlib_reportFatalErrorInFile + 115
5    libswiftCore.so                    0x00007f88d79b1270 _assertionFailure(_:_:file:line:flags:) + 523
7    libFoundation.so                   0x00007f88d87aefc0 _NSCFConstantString.__deallocating_deinit + 25
10   libFoundation.so                   0x00007f88d8b089e0 CFRelease + 73
15   libFoundation.so                   0x00007f88d8b09a20 _CFDeinit + 106
16   libFoundation.so                   0x00007f88d87a56d0 _NSCFDictionary.deinit + 47
17   libFoundation.so                   0x00007f88d87a57f0 _NSCFDictionary.__deallocating_deinit + 25
20   libFoundation.so                   0x00007f88d8b089e0 CFRelease + 73
24   libFoundation.so                   0x00007f88d8b09a20 _CFDeinit + 106
25   libFoundation.so                   0x00007f88d879bd40 _NSCFArray.deinit + 47
26   libFoundation.so                   0x00007f88d879be60 _NSCFArray.__deallocating_deinit + 25
28   libFoundation.so                   0x00007f88d89a0630 NSURLComponents.queryItems.getter + 756
32   libFoundation.so                   0x00007f88d84f8730 _MutableHandle.map<A>(_:) + 189
33   libFoundation.so                   0x00007f88d8aec020 URLComponents.queryItems.getter + 120
```

I'm not sure how it would since `_kCFURLComponentsNameKey as! String` should just be used to compute a hash but perhaps I'm misunderstanding something.